osutil/mountprofile_linux_test.go

```diff
@@ -78,6 +78,14 @@ name#-1 dir#-1 type#-1 options#-1 1 1 # inline comment
 	})
 }
 
+func (s *profileSuite) TestLoadMountProfileText(c *C) {
+	p, err := osutil.LoadMountProfileText("tmpfs /tmp tmpfs defaults 0 0")
```

Comment:
The test should probably use a multi line string to ensure the input is parsed line by line.